packages/runtime-fetch/lib/DownloadRuntime.js

```diff
@@ -19,7 +19,7 @@ const cacheListProvider = require('@ampproject/toolbox-cache-list');
 const crossFetch = require('cross-fetch');
 const fse = require('fs-extra');
 const https = require('https');
-const log = require('@ampproject/toolbox-core').log.tag('AMP Download Runtime');
+const log = require('@ampproject/toolbox-core').log.tag('Runtime Fetch');
```

Comment:
nit: Tag doesn't match tool name; this is the only place "Runtime Fetch" is user facing.

Reply:
renamed to `Runtime Download`